src/pymor/algorithms/rand_la.py

```diff
@@ -71,8 +72,29 @@ def __init__(self, A, range_product=None, source_product=None, A_adj=None,
             A_adj = AdjointOperator(A, range_product=range_product, source_product=source_product)
 
         self.__auto_init(locals())
-        self.R, self.estimator_last_basis_size, self.last_estimated_error = None, 0, np.inf
+        self.Omega = A.range.empty()
+        self.T = None
+        self.estimator_last_basis_size, self.last_estimated_error = 0, np.inf
         self.Q = [A.range.empty() for _ in range(power_iterations+1)]
+        self.R = [np.empty((0,0)) for _ in range(power_iterations+1)]
+
+    def _draw_samples(self, num):
+        # returns samples of the range of A
```

Comment:
Since this is intended to be overwritten, a proper docstring would be nice.
Also, the samples are in the source of A, right?